testsuite/test_serverclass.py

```diff
@@ -0,0 +1,267 @@
+#
+# (C) Copyright 2018 InovaDevelopment.com
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+# Author: Karl  Schopmeyer <inovadevelopment.com>
+#
+
+"""
+Test for the _server class in pywbem that uses the mock support to exercise
```

Comment:
There is no "_server" class. Either this test module as the scope of the "_server module" or the "WBEMServer class". I suppose it is the former.

Reply:
Since the only thing in the_server module is the WBEMServer class it is probably more logical to make it the WBEMServer class

Changed the name of the test file.

Reply:
Changed name in doc string and also changed name to test_wbemserverclass.py